### ISSUE TITLE
Nan appears during training

### ISSUE THREAD

**Issue by HospitableHost** [garvita-tiwari/PoseNDF#21]
I tried to train Pose-NDF, but after some epochs of training, the problem of Nan always appeared. I believe you should have encountered the same problem. Would you like to tell me how to solve this problem? Thank you very much!
![image](https://user-images.githubusercontent.com/44576391/221495820-491d9dbf-6f6b-49f1-800b-e5fadd1285c2.png)
The triangle in the figure represents nan.

**Comment by garvita-tiwari** [comment]
We figured that this occurs due to weight norm layer of pytorch: https://github.com/pytorch/pytorch/issues/19126

Train without weight normalization
